perun-openapi/openapi.yml

```diff
@@ -7806,6 +7806,37 @@ paths:
                 sponsor: { type: integer }
                 namespace: { type: string }
 
+  /json/membersManager/setSponsoredMember:
+    post:
+      tags:
+        - MembersManager
+      operationId: setSponsoredMember
+      summary: Sets the existing user as the sponsored user.
```

Comment:
```suggestion
      summary: Sets the existing user as the sponsored member.
```

Reply:
I would maybe use something like `Sponsor the given member in the given vo` or `Creates a sponsored membership for the given user`